tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -718,4 +718,4 @@ jobs:
       condition : 'succeeded'
   - task: mspremier.PostBuildCleanup.PostBuildCleanup-task.PostBuildCleanup@3
     displayName: 'Clean Agent Directories'
-    condition: always()
\ No newline at end of file
+    condition: always()
```

Comment:
Looks like you didn't change anything in this file. So please exclude it from your commit.

Reply:
Done.

Reply:
Done.